test/cmd/frontendclient/main.go

```diff
@@ -194,7 +194,7 @@ func waitForResults(resultsChan chan frontend.Player, stream frontend.Frontend_G
 				break
 			}
 			log.Println("Error encountered")
-			log.Println("Error reading stream for GetAssignments(_) = _, %v", err)
+			log.Printf("Error reading stream for GetAssignments(_) = _, %v", err)
```

Comment:
nit/curious. I haven't seen placeholders like the "*_*" before to visualize the input and right-side? Is that a standard pattern/does it work with the logging library in some interesting way? (I realize it's carry-over from the previous code... just curious :) )

Reply:
Not sure probably not worth looking into just because this code is probably going to die.